lib/__tests__/standalone.test.js

```diff
@@ -298,13 +297,13 @@ describe("standalone with deprecations", () => {
   let ruleDefinitionsStub, results, output
 
   beforeEach(() => {
-    ruleDefinitionsStub = sinon.stub(ruleDefinitions, "block-no-empty", () => {
+    ruleDefinitionsStub = Object.assign({}, ruleDefinitions, { "block-no-empty": jest.fn(() => {
```

Comment:
I think the issue here is that the mocked function isn't being applied to the required `ruleDefinitions` variable. You could try creating a [manual mock](https://facebook.github.io/jest/docs/manual-mocks.html) in `lib/__mocks__/rules.js`, then use `jest.mock(ruleDefinitions)`.

You might also need to move this "standalone with deprecations" test out to a new `standalone-deprecations.test.js` file so that the mocked `ruleDefinitions` doesn't interfere with the other tests in this file.

Let me know if you've any questions.